test/integration/testsForApiVersion1.spec.js

```diff
@@ -0,0 +1,110 @@
+const { setupForApiVersion1 } = require('./util')
+const evrythng = require('../../dist/evrythng.node')
+
+describe('evrythng.js for apiVersion = 1', function () {
+  let settings = evrythng.setup({ apiVersion: 1, geolocation: false })
+
+  before(async () => {
+    settings = await evrythng.setup({ apiVersion: 1, geolocation: false })
+    await setupForApiVersion1(settings.apiUrl)
+  })
+
+  describe(`as Application for ${settings.apiVersion}`, () => {
```

Comment:
Suggest you destructure `settings` properties used here repeatedly to make it a little cleaner:
```js
const { apiUrl, apiVersion } = settings
```
Then you can just do
```js
require('./entity/actions.spec')(scope, apiUrl)
```
etc in these two files

Reply:
fixed, thanks